docs/triggertemplates.md

```diff
@@ -51,6 +51,8 @@ spec:
 
 Similar to [Pipelines](https://github.com/tektoncd/pipeline/blob/master/docs/pipelines.md),`TriggerTemplates` do not do any actual work, but instead act as the blueprint for what resources should be created.
 Any parameters defined in the [`TriggerBinding`](triggerbindings.md) are passed into to the `TriggerTemplate` before resource creation.
+If the namespace is omitted, it will be resolved to the `TriggerTemplate` namespace.
```

Comment:
In my opinion, it makes most sense if this is the `EventListener`'s namespace (even though at the moment these are all in the same namespace)

Reply:
Also, thanks for keeping the docs up-to-date! 💯